### PR TITLE
Add blog (#2)

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Nowosad** [comment]
@Robinlovelace I think it should work now (I think the repo use GHA for rendering, not netlify
)

**Comment by Nowosad** [comment]
@Robinlovelace I think you may merge it (we can fix any issues later)

File: _freeze/post/2023/ogh23/index/figure-html/unnamed-chunk-15-1.png
[the remark this answers is not in the document]
Comment (Nowosad) [reply _freeze/post/2023/ogh23/index/figure-html/unnamed-chunk-15-1.png:1]: I've done quarto preview, but maybe quarto render would be better. Not sure

**Comment by Nowosad** [comment]
It's online -- https://geocompx.org/post/2023/ogh23/

**Comment by Robinlovelace** [comment]
:rocket:

**Comment by michaeldorman** [comment]
Excellent, thanks @Robinlovelace , @Nowosad , @anitagraser !

**Pull request merged**